client/src/containers/tendencies/index.tsx

```diff
@@ -48,31 +45,23 @@ const Tend: React.FC = () => {
         data: tendencies.map((tendency) => ({
           date: moment(tendency.date).format('YYYY-MM'),
           amount: Number(tendency.amount.toFixed(2)),
-          coefficient: Number(tendency.coeficient.toFixed(5))
-        }))
+          coefficient: Number(tendency.coeficient.toFixed(5)),
+        })),
       });
     }
-  }, [tendencies]);
+  }, [tendencies.length]);
```

Comment:
kas čia per keistas change?

Reply:
Paliekant tik tendencies, meta errorą "Maximum update depth exceeded. This can happen when a component calls setState inside useEffect, but useEffect either doesn't have a dependency array, or one of the dependencies changes on every render.", tendencies.length sufixino errorą.

Reply:
I'll look into it, tiesiog randomly pakeist dependancy nėra geras sprendimas, "kad nemestų erroro".

Reply:
![image](https://user-images.githubusercontent.com/47576481/83399672-44419b80-a40a-11ea-8aae-d419b368f663.png)
literally meta priešingą warningą.

Reply:
radau problemą, tau reik pakeist, kad tendencies būtų ne dictionary o array, kaip ir esu minėjęs anksčiau, Object.keys() kai darai kiekvieną kart atrodo, kad pasikeitęs yra dependency.